### PR TITLE
pherry: config transaction longevity

### PR DESCRIPTION
**Pull request by kvinwang**
tested:
- [x] e2e

### CODE REVIEW

**Comment by kvinwang** [comment]
tested:

- [x] e2e

File: standalone/pherry/src/msg_sync.rs [hunk 1]
```diff
@@ -42,6 +58,30 @@ impl<'a> MsgSync<'a> {
 
         self.maybe_update_signer_nonce().await?;
 
+        let era = if let Some(longevity) = self.longevity {
+            let cur_block = crate::get_block_at(self.client, None).await?.0;
```
Comment (h4x3rotab) [inline standalone/pherry/src/msg_sync.rs:62]: We can get the header only.

Comment (kvinwang) [reply standalone/pherry/src/msg_sync.rs]: Done

**Review by h4x3rotab — APPROVED** [review]
LGTM

**Pull request merged**